common/content/buffer.js

```diff
@@ -415,6 +416,18 @@ const Buffer = Module("buffer", {
     set textZoom(value) { Buffer.setZoom(value, false); },
 
     /**
+     * @property {number} The current browser's zoom level, as a
+     *     percentage with 100 as 'normal'.
+     */
+    get zoomLevel() {
```

Comment:
Not really sure why, but zoomlevel seems to be represented as 3 digits wide even for < 100% zoom and results in `( 80%)` vs `(100%)`. Maybe not something with your changes, but the `ZoomManager`?